src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,18 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
-}
\ No newline at end of file
+    } 
+    
+    /**
+     * Returns a String array after removing white spaces preceding or
+     * succeeding any string in that array
+     */
+    public static String[] trim(String[] stringsToTrim) {
+        String stringsAfterTrim[] = new String[stringsToTrim.length];
+        int m = 0;
```

Comment:
I mean the variable `m`. Shouldn't we use `i` or more meaningful variable name instead?